frontend/www/js/omegaup/components/arena/ContestPractice.vue

```diff
@@ -52,8 +53,11 @@
     </template>
     <template #arena-clarifications>
       <omegaup-arena-clarification-list
+        :problems="problems"
         :clarifications="currentClarifications"
+        :is-admin="contestAdmin"
         :in-contest="true"
+        @new-clarification="(target) => $emit('new-clarification', target)"
```

Comment:
```suggestion
        @new-clarification="(request) => $emit('new-clarification', request)"
```

nit: `target` deberíamos reservarlo para el componente (si es que se manda). en este caso, el parámetro es de tipo `ClarificationRequest` , así que podemos llamarlo `request`.

Reply:
Listo. Esto lo voy a agregar en algún documento de la wiki porque nos va a ayudar a recordarlo.